### PR TITLE
Fix UI tooltips

### LINKED ISSUE #998: Tool tip strings for map starting positions do not show in menu
Thyme does not show tool tip strings in menu.

To reproduce hover over starting position in Skirmish Match Room.

Issue happens with 6e091f516ec89e852a34784a976a5db32c3c42a1 (but is not introduced by it).

## Original

shot_20230926_225931_1

## Thyme

shot_20230926_225820_1

### LINKED ISSUE #1138: Some UI tooltips don't appear
Tooltips on checkbox and combobox hover don't appear. Listboxes work fine.
First noticed in bd8781e93e8c27cec6e346ac55a89efbb1c90665

### Original
image

### Thyme
image

### PR DESCRIPTION
**Pull request by psykana**
Radio button, checkbox, slider gadgets set their tooltips here: https://github.com/TheAssemblyArmada/Thyme/blob/b7a373aa62cca0b30f96284a9cbb3e3da9f31545/src/game/client/gui/gamewindowmanager.cpp#L769-L774
Since `GameWindow` ctor incorrectly initialized `window->m_tooltip` to a stub function, Thyme never got past the first check.

That didn't fix comboboxes though, which led me to `GameWindowManager::Is_Hidden()`.

Fixes #1138
Fixes #998

### CODE REVIEW

**Review by xezon — COMMENTED** [review]
Changes in `GameWindowManager::Is_Hidden` look logical to me. Change in constructor I do not know.

File: src/game/client/gui/gamewindow.cpp [hunk 1]
```diff
@@ -52,7 +52,6 @@ GameWindow::GameWindow() :
     Win_Set_Draw_Func(g_theWindowManager->Get_Default_Draw());
     Win_Set_Input_Func(g_theWindowManager->Get_Default_Input());
     Win_Set_System_Func(g_theWindowManager->Get_Default_System());
-    Win_Set_Tooltip_Func(g_theWindowManager->Get_Default_Tooltip());
```
Comment (jonwil) [inline src/game/client/gui/gamewindow.cpp:55]: Correct change here is Win_Set_Tooltip_Func(nullptr);

**Comment by jonwil** [comment]
Changes in GameWindowManager::Is_Hidden are correct.

**Review by jonwil — COMMENTED** [review]
Looks good to me, approved.

**Comment by psykana** [comment]
huh? something wrong with CI? I don't think I broke that...

**Review by xezon — APPROVED** [review]

**Pull request merged**